### PR TITLE
give outstanding reqs report url param to pre-select state

### PR DESCRIPTION
**Pull request by iaincrawf**
Could now give links for finance, orders etc ie ?state=FINANCE%20WAIT and loads relevant data straight away

+ fix description typo for dept search on req ut

### CODE REVIEW

**Review by richardiphillips — CHANGES_REQUESTED** [review]
looks good but useLocation is better

File: src/Service.Host/client/src/components/reports/OutstandingPoReqsReport.js [hunk 1]
```diff
@@ -29,7 +30,21 @@ function OutstandingPoReqsReport() {
         dispatch(purchaseOrderReqStatesActions.fetch());
     }, [dispatch]);
 
-    const [reqState, setReqState] = useState(null);
+    const options = useMemo(() => queryString.parse(window.location.search) || {}, []);
```
[the remark this answers is not in the document]
Comment (iaincrawf) [reply src/Service.Host/client/src/components/reports/OutstandingPoReqsReport.js]: Ah yeah forgot about that, switched now. Took me a ridiculous amount of time until I finally read some documentation for useLocation

**Review by richardiphillips — APPROVED** [review]
looks good. would it not be safer to just pass search into the useEffect maybe rather than going through another static variable?

**Comment by iaincrawf** [comment]
> looks good. would it not be safer to just pass search into the useEffect maybe rather than going through another static variable?

It could be, I just put it in a variable because I use it again to set the reqState to it

**Pull request merged**